### DISCUSSION TITLE
Rich text editor

### DISCUSSION THREAD

**Discussion by clinton-mwachia** [chakra-ui/chakra-ui#8273] [Ideas] [answered]
### Description

I am requesting the rich-text editor feature so that I or any other developer can access the functions without having to install additional libraries to get the editor only.

### Problem Statement/Justification

Hello Chakraui team, I am grateful for such an excellent library. Chakraui is my go-to library when creating dashboards or any other UI project. I am currently working on a project that requires a rich-text editor like [this](https://primereact.org/editor/), for me to use this component, I have to install an entire library. 

This is not good because I will end up with a large bundle, as a developer, I like to achieve the most with as few libraries as possible so having this feature will be a + for us all. thanks.

### Proposed Solution or API

https://primereact.org/editor/

**Comment by DezyNation** [comment, marked as answer]
An inbuilt rich text editor will be amazing!

**Comment by Onyelaudochukwuka** [comment]
Is this being worked on?

**Comment by clinton-mwachia** [comment]
I don't think so

**Comment by zurghani** [comment]
I think this is a much needed feature.. 
what I like about Chakra is the easy out of box experience, I used it for many of projects and always finding what I need already there.
However, I thing adding an Editor for  "Rich Text" is going to be wonderful, instead of using another lib for it.

**Reply by clinton-mwachia** [reply]
Sure, I love it as well, it will be even better if they can include these other features.

**Comment by jcombs-cb** [comment]
+1

**Comment by webian** [comment]
+1

**Comment by Minhaj-beep** [comment]
+1

**Comment by thiago-roberto** [comment]
+1

**Comment by Adebesin-Cell** [comment]
Hey! Great news, the Rich Text Editor is now part of Chakra UI as of v3.31.

- Blog post: https://chakra-ui.com/blog/chakra-3.31-rich-text-editor
- Docs: https://chakra-ui.com/docs/components/rich-text-editor

Give it a try and let us know how it goes 🙌
